sdk/dotnet/Iam/ServerCertificate.cs

```diff
@@ -229,12 +247,6 @@ public static ServerCertificate Get(string name, Input<string> id, ServerCertifi
 
     public sealed class ServerCertificateArgs : Pulumi.ResourceArgs
     {
-        /// <summary>
-        /// The Amazon Resource Name (ARN) specifying the server certificate.
-        /// </summary>
-        [Input("arn")]
-        public Input<string>? Arn { get; set; }
```

Comment:
What happened here?

Reply:
An ARN should never be an input argument - this is expected but I am not sure why it was ever there in the first place